.github/styles/kong/Terms.yml

```diff
@@ -12,7 +12,7 @@ swap:
   dbless: DB-less
   file name: filename
   hashicorp: HashiCorp
-  kong: Kong
+  '(?:# )kong(?: )': Kong
   # '(?:kubernetes|k8s)': Kubernetes
```

Comment:
We should be able to uncomment the kubernetes/k8s line, as long as we add the same ignores as for `kong`.